### PR TITLE
Added: Wiring, slider, responsiveness

### PR DESCRIPTION
**Pull request by hussiantabha**
Added wiring to all the elements and added slider and made it responsive

### CODE REVIEW

File: files/signup.html [hunk 1]
```diff
@@ -27,7 +27,7 @@
                 <span class="badge">2</span>
             </div></a>
         </div>
-        <div class="nav-hamburger"> <i class="fas fa-bars "></i></i></div>
+        <div class="nav-hamburger"> <i class="fas fa-bars hamburger-icon"></i></i></div>
```
Comment (Kothalkarpriya) [inline files/signup.html]: you used two</i> closing tags for i

File: files/wishlist.html [hunk 2]
```diff
@@ -27,7 +27,7 @@
                 <span class="badge">2</span>
             </div></a>
         </div>
-        <div class="nav-hamburger"> <i class="fas fa-bars "></i></i></div>
+        <div class="nav-hamburger"> <i class="fas fa-bars hamburger-icon"></i></i></div>
```
Comment (Kothalkarpriya) [inline files/wishlist.html]: here also

File: index.html [hunk 3]
```diff
@@ -27,28 +27,34 @@
...
-            <h3>Exotic</h3>
-        </div>
-        <div class="categories">
-            <img src="../assets/flavored.png" class="categories-img">
-            <h3>Day to Day Tea</h3>
-        </div>
+        </div></a>
+        <a href="./files/products.html">
+            <div class="categories">
+                <img src="../assets/exotic.png" class="categories-img">
```
Comment (Kothalkarpriya) [inline index.html]: use / before closing img tag as it is self closing tag

File: index.html [hunk 4]
```diff
@@ -27,28 +27,34 @@
...
+                <h3>Exotic</h3>
+            </div>
+        </a>
+        <a href="./files/products.html">    
+            <div class="categories">
+                <img src="../assets/flavored.png" class="categories-img">
```
Comment (Kothalkarpriya) [inline index.html]: here as well

**Review by Kothalkarpriya — CHANGES_REQUESTED** [review]
Do make the suggested changes, and then you can merge

File: css/product.css [hunk 5]
```diff
@@ -31,6 +31,16 @@
 .filter-category span {
   margin-left: 0.5rem;
```
Comment (atharvabhanage02) [inline css/product.css:32]: As Vikram suggested there is no need to change px to rem if its below 10px

**Review by atharvabhanage02 — APPROVED** [review]
Just one suggestions
Else the code looks clean and well structured

File: css/product.css (hunk 5 above)
Comment (hussiantabha) [reply css/product.css:32]: Got it

File: css/product.css [hunk 6]
```diff
@@ -31,6 +31,16 @@
...
 }
+.filter-slider {
+  width: 80%;
+  background-color: aqua;
```
Comment (Priya-Patel31) [inline css/product.css:36]: don't use colors  directly

File: files/products.html [hunk 7]
```diff
@@ -27,13 +27,21 @@
...
     <aside class="filter-sidebar">
     <section class="filters">
         <div>
-            <input type="text" class="input-text-search" placeholder="search Product">
+            <input type="text" class="input-text-search" placeholder="search Product" >
+        </div>
+        <div>
+            <input type="range" min="1" max="100" value="50" class="filter-slider">
+            <div class="slider-span">
+                <span>0</span>
```
Comment (Priya-Patel31) [inline files/products.html:41]: refactor : don't use span tag as they are non-semantic

File: files/products.html [hunk 8]
```diff
@@ -27,13 +27,21 @@
...
         </div>
-        <div class="nav-hamburger"> <i class="fas fa-bars "></i></i></div>
+        <div class="nav-hamburger"> <i class="fas fa-bars hamburger-icon"></i></i></div>
         </div>
     </nav>
     <aside class="filter-sidebar">
     <section class="filters">
         <div>
-            <input type="text" class="input-text-search" placeholder="search Product">
+            <input type="text" class="input-text-search" placeholder="search Product" >
```
Comment (Priya-Patel31) [inline files/products.html:36]: fix : type should be search rather than text as it is a search bar

File: files/products.html [hunk 9]
```diff
@@ -27,13 +27,21 @@
                 <span class="badge">2</span>
             </div></a>
         </div>
-        <div class="nav-hamburger"> <i class="fas fa-bars "></i></i></div>
+        <div class="nav-hamburger"> <i class="fas fa-bars hamburger-icon"></i></i></div>
```
Comment (Priya-Patel31) [inline files/products.html:30]: fix : remove these two <i> closing tag

**Review by Kothalkarpriya — APPROVED** [review]
good to go

**Review by Priya-Patel31 — APPROVED** [review]
Make these changes as they are necessary . Else everything looks good .

**Pull request merged**